frontend/src/metabase/components/SaveQuestionForm/SaveQuestionForm.tsx

```diff
@@ -75,7 +74,7 @@ export const SaveQuestionForm = ({
         />
       )}
       {values.saveType === "create" && (
-        <div className={CoreStyles.overflowHidden}>
+        <div>
```

Comment:
I'm not really sure why this CSS was needed. It's cutting off the outline of the input focus so I removed it.

Before
![CleanShot 2025-01-03 at 10 21 31@2x](https://github.com/user-attachments/assets/de8f9ba7-0262-41da-9ad0-e5734b123233)

After
![CleanShot 2025-01-03 at 10 21 12@2x](https://github.com/user-attachments/assets/1ba9c3e8-a7af-4a8f-8504-9761258e7980)